### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by MansiM578**

### CODE REVIEW

File: src/components/details/RecentSlide.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            <Box key={item.id} textAlign="center" sx={{ padding: "25px 15px" }}>
+              <Image src={item?.image} alt="image1" />
+              <Text sx={{ fontWeight: 600, color: "#212121" }}>
+                {item.name}
+              </Text>
+              <Text sx={{ color: "green" }}>{item.discount}% off</Text>
+              <Text sx={{ color: "#212121", opacity: ".6" }}>tagline</Text>
+            </Box>
+          ))}
+          {/* <Box textAlign="center" sx={{ padding: "25px 15px" }}>
```
Comment (amishaaggarwal) [inline src/components/details/RecentSlide.tsx]: remove unused code

File: src/components/header/CustomButtons.tsx [hunk 2]
```diff
@@ -33,49 +36,83 @@ const LoginButton = styled(Button)`
...
+  const handleLogout = () => {
+    // Clear user data from local storage
+    localStorage.removeItem("userData");
+    console.log(userData);
+
+    // Dispatch the logout action to update the Redux state
+    dispatch(logout());
+  };
+
+  // function onAuthenticated(): void {
```
Comment (amishaaggarwal) [inline src/components/header/CustomButtons.tsx]: remove unsused code and logs

File: src/components/listItems/ProductList.tsx [hunk 3]
```diff
@@ -39,37 +46,104 @@ const PhoneText = styled(Typography)`
 `;
 
 const Image = styled("img")({
-  width: "auto",
-  height: 150,
-  objectFit: "cover",
+  // width: "auto",
+  // height: 150,
```
Comment (amishaaggarwal) [inline src/components/listItems/ProductList.tsx]: remove unused code

File: src/components/listItems/ProductList.tsx [hunk 4]
```diff
@@ -39,37 +46,104 @@ const PhoneText = styled(Typography)`
...
+`;
+
+const Assured = styled("img")({
+  height: "21px",
+  margin: "0 0 0 8px",
+  width: "77px",
+});
+
+// const Text = styled(Typography)`
+//   font-size: 14px;
```
Comment (amishaaggarwal) [inline src/components/listItems/ProductList.tsx]: remove unused code

File: src/components/listItems/ProductList.tsx [hunk 5]
```diff
@@ -78,34 +152,51 @@ const ProductList: React.FC = () => {
...
+                  onClick={handleProductView}
+                  style={{ textDecoration: "none" }}
+                >
+                  <Model textAlign="center">
+                    <Image src={item.images[0]} alt="phone1" />
+                    <Text1>{item.name}</Text1>
+                    <RateBox>
+                      <ProductRating>
+                        {item?.ratingReviews?.avg_rating} ★
+                        {/* <Rating></Rating> */}
```
Comment (amishaaggarwal) [inline src/components/listItems/ProductList.tsx]: remove unused code

File: src/components/listItems/ProductList.tsx [hunk 6]
```diff
@@ -78,34 +152,51 @@ const ProductList: React.FC = () => {
...
+                      </ProductRating>
+                      <Review component="span">
+                        ({item?.ratingReviews?.rating})
+                      </Review>
+                      <Assured
```
Comment (amishaaggarwal) [inline src/components/listItems/ProductList.tsx:175]: kindly use self closing tags

File: src/components/login/LoginDialog.tsx [hunk 7]
```diff
@@ -48,7 +53,6 @@ const Text = styled(Typography)`
 //   font-size: 14px;
```
Comment (amishaaggarwal) [inline src/components/login/LoginDialog.tsx]: remove unused code

File: src/components/login/LoginDialog.tsx [hunk 8]
```diff
@@ -141,17 +142,130 @@ const LoginDialog: React.FC<props> = ({ open, setOpen }) => {
...
+          form.reset();
+          handleClose();
+        })
+        .catch((error) => {
+          console.log(error);
+        });
+    }
+  };
+
+  // const handleSendCode = async () => {
```
Comment (amishaaggarwal) [inline src/components/login/LoginDialog.tsx]: remove unused code

File: src/pages/ShowPage.tsx [hunk 9]
```diff
@@ -71,27 +71,6 @@ const SideGrid = styled(Grid)`
 const Cart: React.FC = () => {
   const cartItems = useSelector((state: RootState) => state.cart.items);
 
-  // const dispatch = useDispatch();
```
Comment (amishaaggarwal) [inline src/pages/ShowPage.tsx:74]: remove unused code

**Review by amishaaggarwal — CHANGES_REQUESTED** [review]

File: src/components/details/RecentSlide.tsx (hunk 1 above)
Comment (MansiM578) [reply src/components/details/RecentSlide.tsx]: Removed

File: src/components/header/CustomButtons.tsx (hunk 2 above)
Comment (MansiM578) [reply src/components/header/CustomButtons.tsx]: Removed

File: src/components/listItems/ProductList.tsx (hunk 3 above)
Comment (MansiM578) [reply src/components/listItems/ProductList.tsx]: removed

File: src/components/listItems/ProductList.tsx (hunk 4 above)
Comment (MansiM578) [reply src/components/listItems/ProductList.tsx]: removed

File: src/components/listItems/ProductList.tsx (hunk 6 above)
Comment (MansiM578) [reply src/components/listItems/ProductList.tsx:175]: self closing ?

File: src/components/login/LoginDialog.tsx (hunk 7 above)
Comment (MansiM578) [reply src/components/login/LoginDialog.tsx]: Removed

File: src/components/login/LoginDialog.tsx (hunk 8 above)
Comment (MansiM578) [reply src/components/login/LoginDialog.tsx]: Removed

File: src/pages/ShowPage.tsx (hunk 9 above)
Comment (MansiM578) [reply src/pages/ShowPage.tsx:74]: removed

**Review by amishaaggarwal — APPROVED** [review]

**Review by amishaaggarwal — APPROVED** [review]

**Pull request merged**